ts/store/actions/persistedPreferences.ts

```diff
@@ -0,0 +1,9 @@
+import { ActionType, createStandardAction } from "typesafe-actions";
+
+export const preferenceFingerprintIsEnabledSave = createStandardAction(
```

Comment:
`preferenceFingerprintIsEnabledSaveSuccess` ?

Reply:
I'll change it.